DashAI/front/src/components/ParameterForm.jsx

```diff
@@ -12,6 +12,12 @@ import * as S from '../styles/components/ParameterFormStyles';
 
 function genYupValidation(yupInitialObj, schema) {
   let finalObj = yupInitialObj;
+  if (Object.prototype.hasOwnProperty.call(schema, 'maximum')) {
```

Comment:
I believe that in modern js you can simplify this operation

```suggestion
  if ('maximum' in schema) {  # or schema.hasOwnProperty('maximum')
```

Reply:
Indeed, refactored all of these in this file